mkdocs.yml

```
@@ -0,0 +1,79 @@
+#
+# Copyright 2024 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+site_name: Pytest Splunk Addon
+site_author: Splunk
+site_url: "https://splunk.github.io/pytest-splunk-addon/"
+edit_uri: "tree/migrating-docs/docs/"
+remote_branch: gh-pages
+
+repo_name: Pytest Splunk Addon
+repo_url: "https://github.com/splunk/pytest-splunk-addon"
+
+markdown_extensions:
+  - toc:
+      permalink: True
+      toc_depth: 3
+  - smarty
+  - fenced_code
+  - sane_lists
+  - codehilite
+  - pymdownx.superfences
+  - pymdownx.snippets
+
+plugins:
+    - mkdocstrings:
+        handlers:
+            python:
+                setup_commands:
+                    - import sys
+                    - sys.path.append('../')
+                selection:
+                    new_path_syntax: true
+                options:
+                  inherited_members: false
+
+theme:
+  name: "material"
+  palette:
+    primary: "black"
+    accent: "orange"
+  features:
+    - content.code.copy
+    - navigation.indexes
+    - navigation.expand
+
+extra_css:
+    - css/extra.css
+
+nav:
+  - Home: "overview.md"
```

```suggestion
  - Home: "index.md"
```

Please rename `overview.md` to `index.md` so when the website opens it does not show 404.

---

done